app/src/main/java/com/aptoide/uploader/apps/view/MyStoreFragment.java

```diff
@@ -80,13 +106,38 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
     storeNameText = null;
     recyclerView.setAdapter(null);
     recyclerView = null;
+    selectionObservable.dispose();
     logoutConfirmation.dismiss();
     logoutConfirmation = null;
     logoutItem = null;
     toolbar = null;
     super.onDestroyView();
   }
 
+  @Override public void resetSelectionState(){
```

Comment:
should this exist? the adapter on showApps should clear the selected list state and use the selectedPublisher to notify the fragment that there are no selected apps anymore. we should discuss this.

Reply:
Fixed